src/rgw/rgw_common.h

```diff
@@ -1297,6 +1297,7 @@ struct req_state {
   /* aws4 auth support */
   bool aws4_auth_needs_complete;
   bool aws4_auth_streaming_mode;
+  bool aws4_auth_presigned_url_bugfix;
```

Comment:
@jmunhoz I'd comment what bugfix this configurable actually fixes, or maybe rename it to .._boto2_compat or something along these lines